src/DynamoPackages/PackageLoader.cs

```diff
@@ -260,11 +262,38 @@ private void TryLoadPackageIntoLibrary(Package package)
                         }
                         catch (LibraryLoadFailedException ex)
                         {
+                            // Managed exception
+                            // We can still try to load other parts of the package
                             Log(ex.GetType() + ": " + ex.Message);
                         }
+                        catch (DynamoServices.AssemblyBlockedException)
+                        {
+                            blockedAssemblies.Add(assem.Assembly);
+                        }
+                        catch (Exception)
+                        {
+                            failedNodeLibs.Add(assem.Assembly);
+                        }
                     }
                 }
 
+                if (loadedNodeLibs.Count > 0)
+                {
+                    // Try to load any valid node libraries regardless package state
+                    PackagesLoaded?.Invoke(loadedNodeLibs);
```

Comment:
Why did you change the sequence of loading this  by moving it up here from [line 290](https://github.com/DynamoDS/Dynamo/pull/12763/files#diff-7ae21a0e381958737cc115ada84cbd9f96efe3315928cbc44b367967923cf55bL290)?

Reply:
so that I do not have to add it in 2 places (when it succeeds and in the catch blocks)

Reply:
To avoid code duplication...I can put it here (line 283) or in a finally block.